### PR TITLE
Alter -d behaviour to support using module name as the filename

### PR DESCRIPTION
**Pull request by pugnascotia**
As discussed on PR #2, update the -d option to support using module name as the filename.

### CODE REVIEW

**Comment by roman01la** [comment]
Did you test this? I just ran via API with `ListItem` component and it got the following name: `listundefineditem.js`.

**Comment by pugnascotia** [comment]
I did - what command line did you use?

**Comment by roman01la** [comment]
I've used test script here https://github.com/roman01la/html-to-react-components/blob/master/test.js

File: lib/writer.js [hunk 1]
```diff
@@ -14,7 +18,6 @@ function writeToFS(components, options, moduleFileNameDelimiter) {
   options = options || {};
 
   var outPath = options.path || 'components';
-  var delimiter = moduleFileNameDelimiter || '-';
```
Comment (roman01la) [inline lib/writer.js]: To fix wrong filename when calling via API you may want to leave this, but change alternative value to empty string `''`.

File: lib/processor.js [hunk 2]
```diff
@@ -90,9 +90,6 @@ function htmlToReactComponentsLib(tree, options) {
     moduleType = 'es6';
   }
 
-
-  var delimiter = options.moduleFileNameDelimiter || '-';
```
Comment (roman01la) [inline lib/processor.js]: To fix wrong filename when calling via API you may want to leave this, but change alternative value to empty string `''`.

**Comment by pugnascotia** [comment]
Sorry about that - should work now. I'll submit a separate PR for making the Mocha fail more clearly

**Comment by roman01la** [comment]
It works, thanks! 👌

**Comment by roman01la** [comment]
Published as v1.3.0

**Comment by roman01la** [comment]
Right now tests are failing because of overlapping output. I have no idea why is this happening.

**Pull request merged**